core/src/modules/configLoaders/schedule.ts

```diff
@@ -54,6 +53,7 @@ export async function loadSchedule(
     incremental: configObject.incremental,
     recurringFrequency: configObject.recurringFrequency,
     confirmRecords: configObject.confirmRecords,
+    refreshEnabled: configObject.refreshEnabled,
```

Comment:
So for all existing codeConfig files which are missing refreshEnabled, they will get the default value from the model, which is `true`... which I guess is the behavior today